GetCertificate/printer.ts

```diff
@@ -109,13 +139,51 @@ export const printUvci = (
     .when(RecoveryCertificate.is, cr => cr.r[0].ci)
     .exhaustive(); // TODO
 
+/**
+ * Check if Vaccination has ended
+ *
+ * @param v the VaccinationEntry to check
+ * @returns true if process ended, false otherwise
+ */
 export const isVaccinationProcessEnded = (v: VaccinationEntry): boolean =>
   v.dn === v.sd;
 
+/**
+ * Format date value based on language
+ *
+ * @param d the date
+ * @param _lang the preferred language
+ * @returns a formatted date
+ */
 export const formatDate = (d: Date, _lang: PreferredLanguage): string =>
   match(_lang)
     .when(
       l => l === PreferredLanguageEnum.en_GB,
       _ => moment(d).format(DATE_FORMAT_EN)
     )
     .otherwise(() => moment(d).format(DATE_FORMAT_ITA));
+
+/**
+ * The issuer (is) field value used by Italy Healthcare Department
+ */
+export const ITALY_HEALTHCARE_ISSUER = "IT";
+
+/**
+ * Format Certificate Issuer based on its value and preferred language
+ *
+ * @param c the issuer certificate value
+ * @param _lang the preferred language
+ * @returns a formatted Certificate Issuer string
+ */
+export const formatCertificateIssuer = (
+  c: string,
+  lang: PreferredLanguage
+): string =>
+  c !== ITALY_HEALTHCARE_ISSUER
+    ? c
+    : match(lang)
+        .when(
+          l => l === PreferredLanguageEnum.it_IT,
+          _ => "Ministero della Salute"
+        )
+        .otherwise(() => "Ministry of Health");
```

Comment:
What if we rewrite this logic with `fp-ts` pattern? Absolutly not mandatory 😄 
```suggestion
  e
    .fromPredicate<string, string>(
      _ => _ === ITALY_HEALTHCARE_ISSUER,
      identity
    )(c)
    .chain(
      e.fromPredicate(
        _ => _lang === PreferredLanguageEnum.it_IT,
        () => "Ministry of Health"
      )
    )
    .map(() => "Ministero della Salute")
    .fold(identity, identity);```

Reply:
Nice way of building an "if-else" with fp-ts. We'll try this approach in the next chore-branch! TY!